test/lib/ufe/testComboCmd.py

```diff
@@ -395,3 +414,122 @@ def ball35Scale():
 
         self.runTestCombo(expectedTRS)
 
+    def testRotateScalePivotCompensation(self):
+        '''Test that rotate and scale pivot compensation match Maya object.'''
+
+        cmds.file(new=True, force=True)
+        mayaSphere = cmds.polySphere()[0]
+        mayaSpherePath = ufe.PathString.path('|pSphere1')
+        mayaSphereItem = ufe.Hierarchy.createItem(mayaSpherePath)
+
+        import mayaUsd_createStageWithNewLayer
+        mayaUsd_createStageWithNewLayer.createStageWithNewLayer()
+
+        proxyShapePath = ufe.PathString.path('|stage1|stageShape1')
+        proxyShapeItem = ufe.Hierarchy.createItem(proxyShapePath)
+        proxyShapeContextOps = ufe.ContextOps.contextOps(proxyShapeItem)
+        proxyShapeContextOps.doOp(['Add New Prim', 'Sphere'])
+
+        usdSpherePath = ufe.PathString.path('|stage1|stageShape1,/Sphere1')
+        usdSphereItem = ufe.Hierarchy.createItem(usdSpherePath)
+        usdSphereT3d = ufe.Transform3d.transform3d(usdSphereItem)
+
+        # If the Transform3d interface can't handle rotate or scale pivot
+        # compensation, skip this test.
+        if usdSphereT3d.translateRotatePivotCmd() is None or \
```

Comment:
If the Transform3d interface created by the chain of responsibility can't handle rotate and scale pivot compensation, skip the test.